src/hmac.js

```diff
@@ -352,14 +387,12 @@ class AcquiaHttpHmac {
         x_authorization_content_sha256 = willSendBody(body, method) ? CryptoJS.SHA256(body).toString(CryptoJS.enc.Base64) : '',
         signature_base_string_content_suffix = willSendBody(body, method) ? `\n${content_type}\n${x_authorization_content_sha256}` : '';
 
-    parser.href = path;
-
     let site_port = parser.port ? `:${parser.port}` : '',
```

Comment:
We can chain line 378 "let" and line 390 "let" together and make one guinness-record fat let. =D

(Or not, either way is great.)